cmake/FindDependencies.cmake

```diff
@@ -88,14 +88,35 @@ if(CGAL_FOUND)
 endif()
 
 if(DOWNLOAD_ENABLED)
+    # The OpenSSL package in vcpkg seems broken under Windows and leads to
+    # missing certificate verification when connecting to SSL servers. We
+    # therefore use curl[schannel] (i.e., native Windows SSL/TLS) under Windows
+    # and curl[openssl] otherwise.
     find_package(CURL QUIET)
-    find_package(OpenSSL QUIET)
-    if(CURL_FOUND AND OpenSSL_FOUND)
+    set(CRYPTO_FOUND FALSE)
+    if(IS_MSVC AND IS_ARM64)
+        # OpenSSL crashes for ARM64 under Windows. We therefore fall back to
```

Comment:
Is this a temporary issue that we can expect to see fixed soon? If not, why using OpenSSL at all instead of Crypto++ on all platforms? Is it because Crypto++ is more difficult to install? It's unideal to have such complex dependency logic for such a small feature.

Reply:
I would expect this to be fixed eventually. The reason for not defaulting to CryptoPP is that I wanted to keep dependencies minimal on all other platforms. I expect essentially nobody to use colmap under Windows ARM except myself in a VM on my MacBook for debugging.